### DISCUSSION TITLE
How to add more images inside the content

### DISCUSSION THREAD

**Discussion by BabaRagnarok** [nunocoracao/blowfish#1285] [Q&A] [answered]
I added the default image to my articles. Everything OK. I can't figure out how to correctly insert the other images to be included within the content in the post

**Comment by arahmitz** [comment, marked as answer]
You just add them in markdown using proper syntax 
`![alt-text](path to your file "if you want caption, write it in parenthesis")`

Remember to properly slot your images under static folder - I've had problems with it before :D

**Comment by BabaRagnarok** [comment]
Amazing! Now is working thank you
